### PR TITLE
Added infinite sum of gp

### LINKED ISSUE #1299: Sum of Infinite G.P. series in C, C++, Python, Java & Go

### PR DESCRIPTION
**Pull request by MastersAbh**
Fixes #1299 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added implementation for infinite sum gp in 5 languages

#### Languages Used:
- C
- C++
- Python
- Go
- Java

#### Files Added:
- Sum_Of_Infinite_GP.c 
- Sum_Of_Infinite_GP.cpp
- Sum_Of_Infinite_GP.py
- Sum_Of_Infinite_GP.go
- Sum_Of_Infinite_GP.java

### CODE REVIEW

File: Geometric_Progression/Sum_Of_Infinite_GP.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'''
+        SUM OF INFINITE GEOMETRIC PROGRESSION
```
Comment (jainaman224) [inline Geometric_Progression/Sum_Of_Infinite_GP.py]: Fix Indentation.

File: Geometric_Progression/Sum_Of_Infinite_GP.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        The nth term of an G.P. is defined as (a * r^(n-1))
+        where a is the first element of the G.P.
+        r is the common ratio of the G.P.
+        n is the number of term.
+
+        The Sum of infinite series of a G.P. is defined as :
+        a / (1 - r), where -1 < r < 1
+*/
+
+#include <math.h>
```
Comment (somya-kapoor) [inline Geometric_Progression/Sum_Of_Infinite_GP.c]: remove space before<

File: Geometric_Progression/Sum_Of_Infinite_GP.c [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+#include <stdio.h>
```
Comment (somya-kapoor) [inline Geometric_Progression/Sum_Of_Infinite_GP.c]: remove space before<

File: Geometric_Progression/Sum_Of_Infinite_GP.cpp [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+        The Sum of infinite series of a G.P. is defined as :
+        a / (1 - r), where -1 < r < 1
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main() {
+    cout << "Enter the First Term of G.P.";
```
Comment (somya-kapoor) [inline Geometric_Progression/Sum_Of_Infinite_GP.cpp]: Remove space around << and >>

**Comment by MastersAbh** [comment]
Done. @jainaman224 Please review.

**Comment by MastersAbh** [comment]
@rishabhgarg25699 please have a look at this PR.

**Pull request merged**